documenting.rst

```diff
@@ -218,11 +218,11 @@ grasp a simple example more quickly than they can digest a formal description in
 prose.
 
 People learn faster with concrete, motivating examples that match the context of
-a typical use case.  For instance, the :func:`str.rpartition` method is better
+a typical use case.  For instance, the ``str.rpartition`` method is better
```

Comment:
The `intersphinx_mapping` setting in https://github.com/python/devguide/blob/master/conf.py#L33 should resolve this and `sys.ps2`. Perhaps Sphinx didn't follow http -> https redirection?